src/lib/mongoBackend/safeMongo.cpp

```diff
@@ -234,8 +234,9 @@ bool moreSafe(const std::auto_ptr<DBClientCursor>& cursor)
 * nextSafeOrError -
 *
 */
-bool nextSafeOrError(const std::auto_ptr<DBClientCursor>& cursor, BSONObj* r, std::string* err)
+bool nextSafeOrError(const std::auto_ptr<DBClientCursor>& cursor, BSONObj* r, std::string* err, const std::string& caller, int line)
 {
+  // FIXME P4: use line in the trace (just "+ line" doesn't work ;)
```

Comment:
If you want it included, must convert to a string:

    char lineV[32];  // we have a macro for int-as-string-size
    snprintf(lineV, sizeof(lineV), "%d", line);

OR
those ostream constructions that I don't at all like ... :-)